build_release/Diff/MediaInfoLib_MP.diff

```diff
@@ -8,15 +8,15 @@ index efb25e7..6bf20ab 100644
 @@ -99,7 +99,7 @@
        <AdditionalIncludeDirectories>../../../Source;../../../Source/ThirdParty/aes-gladman;../../../Source/ThirdParty/hmac-gladman;../../../Source/ThirdParty/sha1-gladman;../../../Source/ThirdParty/sha2-gladman;../../../Source/ThirdParty/tinyxml2;../../../Source/ThirdParty/md5;../../../Source/ThirdParty/base64;../../../../ZenLib/Source;../../../../ZLib;../../../../curl/include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
        <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
-       <PreprocessorDefinitions>MEDIAINFO_MPEGTS_DUPLICATE_NO;MEDIAINFO_LIBCURL_DLL_RUNTIME;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+       <PreprocessorDefinitions>MEDIAINFO_MPEGTS_DUPLICATE_NO;MEDIAINFO_LIBCURL_DLL_RUNTIME;MEDIAINFO_GRAPHVIZ_DLL_RUNTIME;%(PreprocessorDefinitions)</PreprocessorDefinitions>
 -      <MultiProcessorCompilation>true</MultiProcessorCompilation>
 +      <MultiProcessorCompilation>false</MultiProcessorCompilation>
```

Comment:
Why disabling multithreading compilation?

Reply:
Not from my patch https://github.com/MediaArea/MediaArea-Utils/blob/master/build_release/Diff/MediaInfoLib_MP.diff